.github/workflows/greet_first_time_contributor.yml

```diff
@@ -0,0 +1,50 @@
+name: Greet first time contributor
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened, closed]
+
+jobs:
+  check_for_first_interaction:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: zephyrproject-rtos/action-first-interaction@v1.1.2-zephyr
```

Comment:
This should point to a tag (e.g. `v1.1.2-1`) so that arbitrary pushes to the branch in the future does not break this workflow.

Reply:
Great catch, thanks. I just renamed the branch and created a `v1.1.2-zephyr` tag.

Reply:
fixed with latest update